### ISSUE TITLE
Will not show in HA integrations list.

### ISSUE THREAD

**Issue by phillip1991** [gogades/hass-infinitive#10]
I am using Home Assistant OS 7.0 on a Rpi4. I am able to open and download the infinitive integration through HACS but unable to then locate it through HA integrations list. I have tried to clear cache, delete and re-downlaod, restart but no luck. Is there any known issue with this happening? With realizing now this is perhaps my only way to integrate my Infinity HVAC I would love for this to work. Thank you in advance for your time and work on this project.

**Comment by mww012** [comment]
Hey!  This integration is still a yaml-based integration so you’ll need to add the appropriate config in your configuration.yaml file.

I can help you step through that if you need any assistance. Just let me know!

**Comment by phillip1991** [comment]
Thank you for the reply. My issue isnt the yaml configuration. I download
from HACS, restart and clear cache, search for Infinitive in HA
integrations to install but it will not show. Any ideas what could be going
wrong? Most recent HA OS installed on Rpi4.

**Comment by mww012** [comment]
When you say you search for Infinitive in the HA integrations I'm assuming you don't mean in HACS.  The integration won't show up there.  I wrote it long before there was an HA Integrations UI option and the yaml file was how everything was added.  It's installed and will be loaded when you put the configuration in the configuration.yaml file.  At some point I'll update it but life has been a bit chaotic lately so I haven't been able to get to it.

Here's all you need to do:
1. Install the integration via HACS (you've got this handled)
2. Open your configuration.yaml file
3. Input the following config
```
climate:
  platform: infinitive
  host: [Raspberry Pi IP]
  port: 8080
```
4. Reload HA
5. Check for an Infinitive entity in HA

Let me know if you need any help with this and I'll do whatever I can.

**Comment by phillip1991** [comment]
Thank you very much for that clarification. I am rather new to HA and am used to the normal download hacs, HA integration install then yaml config. 
I currently just wanted to be sure I could get the integration installed properly before I went ahead and purchased the Rpi and adpt for the furnace side of things. I set up config with 0.0.0.0 Host just to compile properly. The only error I found was the failed to connect max retries which was expected so that's good. I did not find the entity in the list but I assume it is solely because the integration cannot find the host? Feel a bit more certain now about my ability to get it going so I am going to order the Rpi and try connecting.
